odoo/code_upgrade/17.5-00-tree-to-list.py

```diff
@@ -0,0 +1,43 @@
+import re
```

Comment:
Shouldn't code upgrades be inside `odoo.upgrades`?

Reply:
no we want it inside regular odoo/

Reply:
Maybe rename to `odoo/upgrade_code` so that it is besides the other package?